generated/niscope/session.py

```diff
@@ -226,6 +231,22 @@ class _SessionBase(object):
     Complex (1)
     Default Value: Complex
     '''
+    ddc_direct_register_address = attributes.AttributeViInt32(1151305)
```

Comment:
This attribute is obsolete. Why are you adding back obsolete attributes?
See niScopeObsolete.h. It appears that most of these DDC attributes are specific to the NI 5620. This is a Traditional DAQ obsolete device.